src/acquisition/covidcast/database.py

```diff
@@ -138,6 +138,17 @@ def _reset_load_table_ai_counter(self):
          '1', '1', '1', '1', '1', 1, 1, 1, 1);""")
     self._cursor.execute(f'DELETE FROM epimetric_load')
 
+  def do_analyze(self):
+    """performs and stores key distribution analyses, used for join order and index selection"""
+    # TODO: consider expanding this to update columns' histograms
+    #       https://dev.mysql.com/doc/refman/8.0/en/analyze-table.html#analyze-table-histogram-statistics-analysis
+    self._cursor.execute(
+      f'''ANALYZE TABLE
+          signal_dim, geo_dim,
+          {self.load_table}, {self.history_table}, {self.latest_table}''')
+    output = [self._cursor.column_names] + self._cursor.fetchall()
+    get_structured_logger('do_analyze').info("ANALYZE results: "+str(output))
```

Comment:
does it make sense to make use of structure here, a la the following?
```suggestion
    get_structured_logger('do_analyze').info("ANALYZE results", results=str(output))
```

Reply:
probably?  its mostly cursory though.  it will look like `[('Table', 'Op', 'Msg_type', 'Msg_text'), ('covid.signal_dim', 'analyze', 'status', 'OK'), ('covid.geo_dim', 'analyze', 'status', 'OK'), ('covid.epimetric_load', 'analyze', 'status', 'OK'), ('covid.epimetric_full', 'analyze', 'status', 'OK'), ('covid.epimetric_latest', 'analyze', 'status', 'OK')] ` in the standard/typical case.  i dont know what it means if it does not output that, and i dont know how to test it to make it output anything but that.  if there is some sequence of events that would cause a different output, it would be indicative of a deeper issue or error that we will hopefully be notified of somewhere other than here.